static/src/styles/custom/gamegig2017.styl

```diff
@@ -0,0 +1,199 @@
+@require '../core/index';
+
+body {
+  background: url("../../images/gamegig3000/gamegig-background.jpg") no-repeat center center fixed;
+  background-size: cover;
+  color: white;
+  text-shadow: 0px 0px 10px blue;
+}
+
+a {
+  text-decoration: none;
+  padding: none;
+  --link-color: #fd7cb3;
```

Comment:
What's this syntax?

Reply:
Stylus

Reply:
The hot new thing ;)

Reply:
Is it worth using these over less variables?

Reply:
I honestly don't have an opinion either way, I think it's probably worth discussing with @Pinpickle .

I'm just not knowledgable enough.

Reply:
We got a bunch of tradeoffs here!

For CSS Custom Properties:

- An official thing, so people get to learn something applicable for every project as opposed to those with preprocessors
- You can set them on a per selector basis. On one link I can set `--link-color` to one thing, and on another I can set it to something else and they won't conflict.
- Can set them dynamically in JS or CSS pseudo-classes (like `:hover`) and changes will propagate (+++)
- You can give them values before or after you use them, putting less importance on file ordering

Against:

- Browser support is getting better, but not fully there: https://caniuse.com/#feat=css-variables (notable example is IE 11 but I think this is fine for our userbase)
- We now have another decision to make when making a variable: Should this be in Stylus or in CSS?
- Fair bit more verbose than Stylus
- Have to go "all in" on CSS when using them. So we can't use Stylus operators (have to use `calc`) or colour functions (these are coming to CSS but are basically a dream at the moment)

Reply:
Can we please come to a conclusion on what to do in this instance? The debate is interesting, but I want to get this live ASAP.

Reply:
Probably best to stick with Stylus for now to avoid changing things?

Reply:
Currently I'm using CSS vars I think, so change those to stylus ones yes?

Reply:
That's the idea! To clarify I meant avoiding changing the convention in our codebase